src/AvenueClothing/Views/Membership/LoginForm.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@using AvenueClothing.Controllers
+@using Umbraco.Web
+@model AvenueClothing.Models.LoginViewModel
+
+@if (User.Identity.IsAuthenticated)
+{
+    <p>
+        Welcome, @User.Identity.Name
+    </p>
```

Comment:
Why not leave it as 1 line?
The content is simple.

Reply:
The formatter inside Rider suggests this style. It's not less simple this way IMO.